cves/kernel/CVE-2016-5400.yml

```diff
@@ -75,7 +79,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [Memory Overflow, Memory Leak]
```

Comment:
These should include bug numbers.
https://bugzilla.redhat.com/show_bug.cgi?id=1358184